lib/schemaUtils.js

```diff
@@ -1150,8 +1151,10 @@ module.exports = {
     }
 
     _.forEach(securitySet, (security) => {
+      let helper;
+
       if (_.isObject(security) && _.isEmpty(security)) {
-        helper = {
+        authHelper = {
```

Comment:
Why are we using `authHelper` here specifically if it will anyways get assigned `helper` in line no. 1300.